custom-login/src/Messages.jsx

```diff
@@ -0,0 +1,85 @@
+import { withAuth } from '@okta/okta-react';
+import React, { Component } from 'react';
+import { Container, Header, Icon, Message, Table } from 'semantic-ui-react';
+import Navbar from './Navbar';
+
+export default withAuth(class Profile extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { messages: null, failed: null };
+  }
+
+  componentDidMount() {
+    this.getMessages();
+  }
+
+  async getMessages() {
+    if (!this.state.messages) {
+      try {
+        const accessToken = await this.props.auth.getAccessToken();
+        /* global fetch */
+        const response = await fetch('http://localhost:8000/api/messages', {
```

Comment:
Curious if it makes sense to not hardcode the base URL + port. That way someone could easily update their `.samples.config.json` to their discretion.